tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py

```diff
@@ -100,7 +101,7 @@ def initializeOtaProject(self):
     def generateFactoryImage(self):
         # If this board uses the Amazon FreeRTOS reference bootlaoder, then we want to
         # build and flash the factory image.
-        if self._buildConfig.get('use_reference_bootloader', False):
+        if self._buildConfig.get('use_reference_bootloader', False) and OtaAfrProject.OTA_FACTORY_IMAGE_GENERATOR_PATH is not None:
```

Comment:
"is not None" is not required.

Reply:
Removed all of them in below commit. :-)